packages/contracts/src/build-metadata.json

```
@@ -1,6 +1,6 @@
 {
   "ui": {},
-  "change": "v1.3\n - Removed an unneccessary permission that allowed the Dao to upgrade the plugin, because this is supposed to happens as part of the update itself. The unnecessary permission, which was granted on installation of previous versions, will be automatically removed with the update to this version.\n",
+  "change": "v1.3\n - Removed an unneccessary permission that allowed the Dao to upgrade the plugin, because this is supposed to happens as part of the update itself. The unnecessary permission, which was granted on installation of previous versions, will be automatically removed with the update to this version.\n - Allow the DAO to still mint new tokens ater uninstallation by not revoking the associated permission.",
```

```suggestion
  "change": "v1.3\n - Removed an unneccessary permission that allowed the Dao to upgrade the plugin, because this is supposed to happens as part of the update itself. The unnecessary permission, which was granted on installation of previous versions, will be automatically removed with the update to this version.\n - Allows the DAO to still mint new tokens after uninstallation by not revoking the associated permission.",
```
😳 I mistyped in my last suggestion